### DISCUSSION TITLE
Dosya Yolu Hatası

### DISCUSSION THREAD

**Discussion by alprslnorgnl** [gusanmaz/BMB212-Algorithms-2223#6] [Q&A] [answered]
İlk projede dosya yolunun bulunamaması durumunda readCSV fonksiyonun döndürdüğü hata mesajı yeterli mi yoksa kendimiz ekstra olarak bir hata mesajı yazdırmamız gerekiyor mu?

**Comment by gusanmaz** [comment, marked as answer]
Evet, yeterli. Dosya yolunun bulunamamasi durumunu test etmeyi dusunmuyorum. Yine de dosya okunamiyorsa programinizin dosya okunamadi turunde bir bilgilendirme mesajini ekrana yazmasi guzel olur.

**Reply by alprslnorgnl** [reply]
Teşekkür ederim
